examples/example3/nautobot_adapter.py

```diff
@@ -0,0 +1,167 @@
+import os
+import pynautobot
+
+from diffsync import DiffSync
+from models import Region, Country
+
+NAUTOBOT_URL = os.getenv("NAUTOBOT_URL", "https://demo.nautobot.com")
+NAUTOBOT_TOKEN = os.getenv("NAUTOBOT_TOKEN", "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
```

Comment:
Might be a good idea to add a note in the README about these two environment variables and how you can set them if you want to point to a local instance of Nautobot instead of the public demo instance?